lib/matplotlib/axes/_base.py

```diff
@@ -3403,7 +3403,8 @@ def tick_params(self, axis='both', **kwargs):
         Change the appearance of ticks, tick labels, and gridlines.
 
         Tick properties that are not explicitly set using the keyword
-        arguments remain unchanged unless *reset* is True.
+        arguments remain unchanged unless *reset* is True. For the current
+        style defaults, see `.Axis.get_tick_params`.
```

Comment:
I find this new clause somewhat ambiguous.  Do you really mean "defaults"?  I think maybe this should be "For the current tick parameters see `.Axis.get_tick_params`." if this is needed at all.